code/modules/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -74,7 +74,7 @@
 	progression_minimum = 0 MINUTES
 	progression_maximum = 45 MINUTES
 	progression_reward = list(4 MINUTES, 8 MINUTES)
-	telecrystal_reward = list(1, 2)
+	telecrystal_reward = list(3)
```

Comment:
This is causing too much of an increase in rewards. I don't want people to get massive amounts of telecrystals from secondary objectives, they already start with 20 TC. Especially since you aren't randomizing the amount of TC they're getting, this gives them a very reliable stream of TC.

Decrease this back down to list(1, 2) tbh